src/txn/txn.c

```diff
@@ -800,11 +801,13 @@ __wt_txn_commit(WT_SESSION_IMPL *session, const char *cfg[])
 	WT_TXN_GLOBAL *txn_global;
 	WT_TXN_OP *op;
 	WT_UPDATE *upd;
-	wt_timestamp_t prev_commit_timestamp;
+	wt_timestamp_t candidate_durable_timestamp, prev_commit_timestamp;
+	wt_timestamp_t prev_durable_timestamp;
```

Comment:
I'd probably split these to put the 2 `prev_` together on the same line and leave the `candidate` one solo.

Reply:
955effc